### PR TITLE
Simplify the data table we need to change with each release

### PR DESCRIPTION
**Pull request by nedbat**
Rather than having to enter two strings which are almost the same, have
just a single string, with a possibility for two strings in the rare
times they are needed.

### CODE REVIEW

**Comment by nedbat** [comment]
@edx/devops too much?

**Comment by maxrothman** [comment]
LGTM 👍

**Comment by nedbat** [comment]
@mulby Can you take a quick look at how I tweaked the Vagrantfile?

File: vagrant/release/analyticstack/Vagrantfile [hunk 1]
```diff
@@ -90,6 +86,15 @@ rel = ENV['OPENEDX_RELEASE']
...
+    boxname = openedx_releases[rel][:name]
+    boxfile = openedx_releases[rel].fetch(:file, "#{boxname}.box")
+  else
+    boxname = reldata
+    boxfile = "#{boxname}.box"
+  end
+
   # Creates an edX analyticstack VM from an official release
   config.vm.box     = openedx_releases[rel][:name]
   config.vm.box_url = "http://files.edx.org/vagrant-images/#{openedx_releases[rel][:file]}"
```
Comment (mulby) [inline vagrant/release/analyticstack/Vagrantfile]: I think these two lines need to be updated too.

Comment (nedbat) [reply vagrant/release/analyticstack/Vagrantfile]: Oops, you are right!. Fixed.

**Review by mulby — APPROVED** [review]
LGTM

**Pull request merged**